src/db.py

```diff
@@ -1,17 +1,50 @@
+
+import sqlite3
 from dataclasses import dataclass
 
+
+# queries to create the db tables
+
+
 @dataclass
 class File:
     name: str
     link: str
-    tags: list[str]
+    tags: "list[str]"
```

Comment:
why?

Reply:
The linter was showing an error that's why!

Reply:
which linter?

Reply:
PyFlakes I guess